scrapers_next/me/people.py

```diff
@@ -0,0 +1,328 @@
+from spatula import (
+    URL,
+    CSS,
+    HtmlListPage,
+    HtmlPage,
+    XPath,
+    SelectorError,
+    ExcelListPage,
+)
+from dataclasses import dataclass
+from openstates.models import ScrapePerson
+import regex as re
+
+
+@dataclass
+class PartialPerson:
+    name: str
+    party: str
+    source: str
+
+
+# is this correct?
```

Comment:
looks good to me, the Passamaquoddy seems like a special case & I'm good with marking it independent